testnets/pocket/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "pocket-beta",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Beta TestNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket-beta",
+  "bech32_prefix": "pokt@",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md